src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -139,9 +149,24 @@ export class FilterController extends Observable {
         let isExtra = false;
 
         let filterRow = this.addEmptyFilter(isDefault, isExtra);
+        let selectedRandomVal = group.values[Math.floor(Math.random() * group.values.length)];
```

Comment:
why random?

Reply:
non-aggregatable columns are required to have a value but the API doesn't send any default value for them. so it was either making the first option selected in default or a random one. Mila told me to make it random

Reply:
@jbothma random doesn't make sense to me - I think it should be the first element in the last. What do you think?

Reply:
@emre2038 please change this to be the first element rather than random

Reply:
done